pyproject.toml

```
@@ -4,7 +4,7 @@ build-backend = "setuptools.build_meta"
 
 [tool.setuptools_scm]
 local_scheme = "node-and-date"
-fallback_version = "unknown"
+fallback_version = "9999.9999.9999"
```

A quick search using https://github.com/search?l=TOML&p=1&q=tool.setuptools_scm+fallback_version&type=Code shows a lot of incosistency across projects. But I did notice someone use something like `0.0.0+unknown`. I think it should be compatible with PEP440 following https://peps.python.org/pep-0440/#local-version-identifiers, but need to test first
```suggestion
fallback_version = "0.0.0+unknown"
```

---

My initial thought is a version string like `0.0.0`, but my concern is, if someone develops a package that requires PyGMT greater than a specific version (for example, checking `pygmt.__version__ >= "0.6.0"`), he won't be able to use the PyGMT with a version string `0.0.0` or `0.0.0+unknown`.

---

It works well. Marking the PR as ready for review.